src/app/components/feed/CreateFeed.js

```diff
@@ -0,0 +1,218 @@
+            placeholder={placeholder}
+            label={<FormattedMessage
+              id="createFeed.titleLabel"
+              defaultMessage="Title"
+              description="Label for the shared feed title input"
+            />}
+            helpContent={<FormattedMessage
+              id="createFeed.titleHelper"
+              defaultMessage="Great shared feed names are short, memorable, and tell your audience the focus of the media"
+              description="Title input helper text"
+            />}
+          />
+        )}
+      </FormattedMessage>
+      <FormattedMessage
+        id="createFeed.descriptionPlaceholder"
+        defaultMessage="Give this shared feed an optional description."
+        description="Placeholder text for feed description field"
+      >
+        { placeholder => (
+          <TextArea
+            placeholder={placeholder}
+            label={<FormattedMessage
+              id="createFeed.descriptionLabel"
+              defaultMessage="Description"
+              description="Label for a field where the user inputs text for a description to a shared feed"
+            />}
+          />
+        )}
+      </FormattedMessage>
+    </div>
+    <div className={styles.createFeedCard}>
+      <div className="typography-subtitle2">
+        <FormattedMessage
+          id="createFeed.feedContentTitle"
+          defaultMessage="Feed content"
+          description="Title of section where a list can be selected as the content of the feed"
+        />
+      </div>
+      <FormattedMessage
+        id="createFeed.feedContentBlurb"
+        defaultMessage="Select a filtered list of fact-checks from your workspace to contribute to this shared feed. You will be able to update this list at any time."
+        description="Helper text for the feed content section"
+      />
+      <FormattedHTMLMessage
+        id="createFeed.feedContentBlurb2"
+        defaultMessage="<strong>Note:</strong> Your list must contain <strong>published fact-checks</strong> in order to be part of this shared feed."
+        description="Helper text for the feed content section"
+      />
+      <Select
+        label={<FormattedMessage id="createFeed.select" defaultMessage="Select list…" description="Label for shared feed list selector" />}
+        className={styles.select}
+        iconLeft={<ListIcon />}
+        helpContent={<FormattedMessage id="createFeed.selectHelper" defaultMessage="Fact-check title, summary,  and URL will be shared with the feed. Learn more" description="Helper text for shared feed list selector" />}
+      >
+        <option value="bli">bla</option>
+      </Select>
+    </div>
+    <div className={styles.createFeedCard}>
+      <div className="typography-subtitle2">
+        <FormattedMessage
+          id="createFeed.publishTitle"
+          defaultMessage="Publish"
+          description="Title of the section where the publishing preferences are set"
+        />
+      </div>
+      <FormattedMessage
```

Comment:
we will eventually put in a link to a help article or direct to the marketplace, might want to build in a temp link now?